examples/docker/modin-ray/build-docker-image.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+cd "`dirname \"$0\"`"
+
+docker build -t modin-ray .
```

Comment:
if I'm right this is the _only_ different line from `build-docker-image.sh` in omnisci example - I suggest to make a common shell script and call it with a parameter "image name" here :)

Reply:
Text messages are different because image name is different and script names are different. Is it different enough to keep files separate?